src/Makevars

```diff
@@ -1,4 +1,2 @@
-
-## We want C++11 as it gets us 'long long' as well
-## If we uncomment this we fall back to C++98 and it is all tears
-CXX_STD = CXX11
+## Use C++11 where available. This ensures 'long long' (a C99 standard)
+PKG_CXXFLAGS=$(CXX1XSTD)
```

Comment:
Pragmatically this has one advantage: If C++11 is not available, the code will still compile. It will throw warnings with `-pedantic`, but for most compilers the code will still work most added C99 support to C++98. This means that if you submit the package to CRAN, it will be available on snow leopard. 

(If you don't care about CRAN, then feel free to ignore this PR, it adds nothing)